packages/client/src/components/Markdown/index.tsx

```diff
@@ -1,13 +1,20 @@
 import React from "react";
-import ReactMarkdown from "react-markdown";
+import ReactMarkdown, { ReactMarkdownProps } from "react-markdown";
 
 import { ListItem } from "../../atoms";
 import Visual from "../Visual";
 import BlockRenderer from "./renderers/BlockRenderer";
 import LinkRenderer from "./renderers/LinkRenderer";
 import ListRenderer from "./renderers/ListRenderer";
 
-export default ({ eventLocation, source }) => (
+type MarkDownProps = {
+  eventLocation: any;
```

Comment:
Why any? Can you replace it with `string`?

Reply:
Nicely spotted